### PR TITLE
update docker hbase port

### PR DESCRIPTION
**Pull request by Sean-Gu**
Refer to: http://hbase.apache.org/book.html#quickstart_fully_distributed

### CODE REVIEW

File: website/_docs/install/kylin_docker.cn.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ docker run -d \
 -p 50070:50070 \
 -p 8032:8032 \
 -p 8042:8042 \
--p 60010:60010 \
+-p 16010:16010 \
 apachekylin/apache-kylin-standalone:3.0.0-alpha2
```
[the remark this answers is not in the document]
Comment (lshmouse) [reply website/_docs/install/kylin_docker.cn.md]: After checking the hbase doc, the default port is changed to 16010.
The file: conf/hbase/hbase-site.xml need be updated

**Comment by lshmouse** [comment]
@Sean-Gu Could you help to update the PR? Thanks~

File: website/_docs/install/kylin_docker.cn.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sean-Gu) [reply website/_docs/install/kylin_docker.cn.md]: Thx, I will update conf/hbase/hbase-site.xml in another PR.

[the remark this answers is not in the document]
Comment (lshmouse) [reply website/_docs/install/kylin_docker.cn.md]: @Sean-Gu  You can just update this PR by by adding a new commit and compress the commites. Please see: https://www.internalpointers.com/post/squash-commits-into-one-git

**Comment by lshmouse** [comment]
@Sean-Gu Would you help to close this PR? Thanks~

**Pull request closed without merging**